docs/001-approval_switch.md

```diff
@@ -0,0 +1,65 @@
+# Approval switch
+The Tangle builds approval of a given message by directly or indirectly attaching other messages in its future cone. Due to different reasons, such as the TSA not picking up a given message before its timestamp is still *fresh* or because its past cone has been rejected, a message can become orphan. This implies that the message cannot be included in the Tangle history since all the recent tips do not contain it in their past cone and thus, it cannot be retrieved during solidification. As a result, it might happen that honest messages and transactions would need to be reissued or reattached.
```

Comment:
Orphan messages still go to the tangle, right? Or do we have a finality check in the "book" box of the data flow?